### PR TITLE
Gradle support

### PR DESCRIPTION
**Pull request by dpisarenko**
Issue: https://github.com/teamed/qulice/issues/339

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, I'll find someone to review this PR soon

**Comment by davvd** [comment]
@carlosmiranda please review, thanks

File: qulice-gradle-plugin/src/main/java/com/qulice/gradle/QulicePlugin.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package com.qulice.gradle;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+
+/**
+ * @author Dmitri Pisarenko (dp@altruix.co)
```
Comment (carlosmiranda) [inline qulice-gradle-plugin/src/main/java/com/qulice/gradle/QulicePlugin.java]: No Javadoc description here?

**Comment by carlosmiranda** [comment]
@dpisarenko Sorry, I'm not that familiar with Gradle. However, I think we're doing the wrong approach here.

The way this is looking, `QulicePlugin` will be one huge class with all the validations being done inside of it. Instead of focusing on `QulicePlugin`, try to create the skeleton of the `qulice-gradle-plugin` project instead. Leave `QulicePlugin` blank for now, and let's create the resources like `pom.xml` (I assume that Gradle support will still be buildable on Maven). Then, add a puzzle asking for the next developer to create a *design* for the validator classes, after which we can jump in to actually coding the validations themselves.

Don't think too low level at the moment, let's focus on design first.

**Comment by dpisarenko** [comment]
@carlosmiranda  Thanks for your feedback. I fixed the issues you reported. Please look again.

File: qulice-gradle-plugin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0"?>
+<!--
+ *
+ * Copyright (c) 2011-2014, Qulice.com
```
Comment (carlosmiranda) [inline qulice-gradle-plugin/pom.xml]: Let's use `2011-2015` dates to be consistent with `LICENSE.txt`

**Comment by carlosmiranda** [comment]
@dpisarenko Thanks for the update, just one more minor comment.

**Comment by dpisarenko** [comment]
@carlosmiranda I fixed the copyright year problem.

**Comment by carlosmiranda** [comment]
@rultor Merge please.

**Comment by yegor256** [comment]
@rultor merge it

**Comment by davvd** [comment]
@carlosmiranda **16 mins** sent to your balance (ID `50519084`), many thanks!. added +16 to your rating, now it is equal to [+1488](http://www.netbout.com/b/35023?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**